src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -31,7 +31,7 @@ const TopicDomainHeader = (props) => {
       </Text>
       <View style={{flexDirection: 'row', alignItems: 'center', marginTop: normalize(1)}}>
         <CommunityIcon
-          color={shouldDisplay ? COLORS.white : undefined}
+          color={shouldDisplay ? COLORS.white2 : undefined}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `CommunityIcon` has been changed from `COLORS.white` to `COLORS.white2` when `shouldDisplay` is true. Ensure that `COLORS.white2` is a valid and defined color in your project's color palette, and it provides the desired visual effect on different screens and themes (light/dark mode).
```diff
- color={shouldDisplay ? COLORS.white : undefined}
+ color={shouldDisplay ? COLORS.white2 : undefined}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->